modules/carto/src/api/layer-map.ts

```diff
@@ -144,8 +155,14 @@ export function getLayer(
   if (config.visConfig?.customMarkers) {
     basePropMap = mergePropMaps(sharedPropMap, customMarkersPropsMap);
   }
-  if (type === 'mvt' || type === 'tileset' || type === 'h3' || type === 'quadbin') {
-    return getTileLayer(dataset, basePropMap);
+  if (
+    type === 'mvt' ||
+    type === 'tileset' ||
+    type === 'h3' ||
+    type === 'quadbin' ||
+    type === 'heatmapTile'
```

Comment:
Not clear how this can work for `'raster'`? Rather than duplicating the types easier to do `if(LAYER_FROM_LAYER_TYPE[type])`

Reply:
Yeah, much nicer ❤️